### ISSUE TITLE
Screen readers do not read off the options for the select menu in the paginator component

### ISSUE THREAD

**Issue by dshank** [tyler-technologies-oss/forge#715]
**Describe the bug:**
Screen readers do not read off the options for the select menu in the paginator component.  You can only find out the value of the option once you've selected it

**To Reproduce:**
1. Go to the paginator component
2. Enable screen reading
3. tab to the select menu for the results per page
4. use the arrow keys to open the menu
5. notice that when you focus on an option, the screen reader does not read out the value

**Expected behavior:**
Focusing on an option will read out the value

**Screenshots:**
<img width="373" alt="image" src="https://github.com/user-attachments/assets/fcb6216d-41a9-4698-a574-c28889703bdf">

**Please complete the following information:**
 - Forge version: 2 & 3
 - I have searched existing issues before creating this report? N
 - Browser: Chrome
 - Platform: Desktop
 - OS: Mac

**Additional context:**
Discussed problem with Kieren directly
